### PR TITLE
Fix Home Owners error handling

### PR DESCRIPTION
**Pull request by dimak1**
*Issue #:* 
- bcgov/entity#17034

*Description of changes:*
- Fixes a specific error scenario in Home Owners table (issue raised by QA)
- Added unit test

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1585-kmnsjibm.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1585-kmnsjibm.web.app

**Review by cameron-eyds — APPROVED** [review]
Looks good!

**Review by RuoxuanPengBC — APPROVED** [review]
Looks great

File: ppr-ui/src/views/newMhrRegistration/HomeOwners.vue [hunk 1]
```diff
@@ -129,7 +129,7 @@
         <v-col cols="9" class="mb-n6 pa-0"></v-col> <!-- Column Spacer -->
         <v-col cols="3" class="mb-n6 pa-0">
           <v-fade-transition>
-            <span v-show="showRemovedAllOwnersMsg" class="error-text fs-12 ml-5">Nothing to delete</span>
+            <span v-if="showRemovedAllOwnersMsg" class="error-text fs-12 ml-5">Nothing to delete</span>
```
Comment (dimak1) [inline ppr-ui/src/views/newMhrRegistration/HomeOwners.vue:132]: @cameron-eyds replaced **v-show** with **v-if**. To remove elements vs hiding with display: none.

**Comment by dimak1** [comment]
Just added another commit with additional fixes and tests.

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1585-kmnsjibm.web.app

**Review by doug-lovett — APPROVED** [review]
Looks good.

**Pull request merged**